src/sdk-reference/cpp/1/getting-started/snippets/init.cpp

```diff
@@ -14,11 +14,11 @@ int main(int argc, char * argv[]) {
     std::cout << "Connected!" << std::endl;
 
     // Freshly installed Kuzzle servers are empty: we first need to create
-    // a data index. The one used in this example is named "nyc-open-data"
+    // a index. The one used in this example is named "nyc-open-data"
```

Comment:
There are other occurences of this error, and they should all be fixed.

```suggestion
    // an index. The one used in this example is named "nyc-open-data"
```